### ISSUE TITLE
SEVERE XSS / RCE Exploit

### ISSUE THREAD

**Issue by LiamDormon** [guillerp8/guille_contextmenu#1]
https://github.com/guillerp8/guille_contextmenu/blob/5160e5a77e11a41bc7494b1fdd2a45b110ef7474/client/client.lua#L48

There's no validation here? As an attacker I could send a post request and execute literally any lua code I want, for example, if a server is using a phone with unsanitised inputs I could send a player a message with some xss code that will allow me to execute code on their client. 

TL;DR
Major remote code execution vulnerability.
